README.md

```diff
@@ -67,13 +67,13 @@ There are two ways how to do it:
 
 2.  Get and unzip the Spline source code:
     ```shell script
-    wget https://github.com/AbsaOSS/spline/archive/release/0.4.0.zip
-    unzip 0.4.0.zip
+    wget https://github.com/AbsaOSS/spline/archive/release/0.5.0.zip
```

Comment:
If develop is your main branch that is shown first when someone opens your project, this can be a bit misleading for the user.

Reply:
100% agree. Actually the content of this README file much better suits the GH pages. I'll rewrite it later, leaving mostly the references to the GH pages and the stuff relevant to the developers.